src/frontend/pages/index.vue

```diff
@@ -1,100 +1,104 @@
-      mise à jour fournit par unpaywall.
-    </p>
-    <v-card v-if="inUpdate">
-      <v-card-title class="justify-center">
-        Status
-      </v-card-title>
-      <v-card-text>
-        <InUpdate />
-      </v-card-text>
-      <v-card-actions @click="show = !show">
-        <v-btn color="orange lighten-2" text>
-          Details
-        </v-btn>
-        <v-spacer />
-        <v-btn icon>
-          <v-icon>{{ show ? "mdi-chevron-up" : "mdi-chevron-down" }}</v-icon>
-        </v-btn>
-      </v-card-actions>
-      <v-expand-transition>
-        <div v-show="show">
-          <Status :status="state" />
-        </div>
-      </v-expand-transition>
+  <section>
+    <v-card class="my-3">
+      <v-toolbar color="secondary" dark flat dense>
+        <v-toolbar-title v-text="$t('home.title')" />
+      </v-toolbar>
+      <v-container>
+        {{ $t("home.intro") }}
+      </v-container>
     </v-card>
-    <v-card v-else>
-      <v-card-title class="justify-center">
-        Status
-      </v-card-title>
-      <v-card-text>
-        <NoInUpdate />
-      </v-card-text>
+
+    <v-card class="my-3">
+      <v-toolbar color="secondary" dark flat dense>
+        <v-toolbar-title v-text="$t('home.metrics')" />
+      </v-toolbar>
+      <v-container>
+        <v-card-title v-text="$t('home.globalMetrics')" />
+        <v-card-text>
+          <v-chip color="grey darken-2" text-color="white">{{ $t('home.referencedResources') }} : {{ metrics.doi }}</v-chip>
+          <v-chip color="grey darken-2" text-color="white">{{ $t('home.openAccess') }} : {{ metrics.isOA }}</v-chip>
+        </v-card-text>
+      </v-container>
+
+      <v-divider />
+      <v-container>
+        <v-card-title v-text="$t('home.openAccessStatus')" />
+        <v-card-text>
+          <v-chip-group active-class="deep-purple accent-4 white--text" column>
+            <v-chip color="#FFC000" text-color="white">
+              <v-icon left color="white">
+                mdi-lock-open
+              </v-icon>
+              gold : {{ metrics.goldOA }}
+            </v-chip>
+
+            <v-chip color="#DD7931" text-color="white">
+              <v-icon left color="white">
+                mdi-lock-open
+              </v-icon>
+              hybrid : {{ metrics.hybridOA }}
+            </v-chip>
+
+            <v-chip color="#DD7931" text-color="white">
+              <v-icon left color="white">
+                mdi-lock-open
+              </v-icon>
+              bronze : {{ metrics.bronzeOA }}
+            </v-chip>
+
+            <v-chip color="#00F765" text-color="white">
+              <v-icon left color="white">
+                mdi-lock-open
+              </v-icon>
+              green : {{ metrics.greenOA }}
+            </v-chip>
+
+            <v-chip color="#BBBBBB" text-color="white">
+              <v-icon left color="white">
+                mdi-lock-open
+              </v-icon>
+              locked : {{ metrics.closedOA }}
+            </v-chip>
```

Comment:
Could be factorised with a `for loop`, but nothing critical

Reply:
i will use a loop